regression-tests/run-tests.sh

```diff
@@ -55,17 +57,21 @@ check_file () {
         git add "$file"
         # ... report the diff ...
         report_diff "$file" \
-            "HEAD" \
             "The $description is not tracked by git" \
-            "$patch_file"
+            "$patch_file" \
+            "HEAD"
         # ... and remove the file from the index
         git rm --cached -- "$file" > /dev/null 2>&1
     else
         # Compare the content with the reference value checked in git
+        # Lines includng Windows paths are excluded from diff
+        # This is necessary due to characters in those paths on GitHub runners
+        # that cause git diff to spuriously flag them
         report_diff "$file" \
-            "--ignore-cr-at-eol" \
             "Non-matching $description" \
-            "$patch_file"
+            "$patch_file" \
+            --ignore-cr-at-eol \
+            --ignore-matching-lines="C:\\\\"
```

Comment:
[Explanation] This option is excluding lines with Windows paths. On the GitHub runners those paths may include characters that seem to cause trouble with `git diff`.
This is hopefully a temporary solution.